core/handlers/v2/simulation_views_v5.go

```diff
@@ -53,6 +53,7 @@ func (this RequestMatcherResponsePairViewV5) GetResponse() interfaces.Response {
 type ResponseDetailsViewV5 struct {
 	Status           int                 `json:"status"`
 	Body             string              `json:"body"`
+	BodyFile         string              `json:"bodyFile"`
```

Comment:
Let's create a new struct for schema changes. For this will be `simulation_views_v6.go`. There are other changes I'd like to add to v6